schema.graphql

```diff
@@ -33,3 +32,82 @@ type AuctionedName @entity {
   bidCount: Int!
   state: AuctionState
 }
+
+type Resolver @entity {
+  id: ID!
+  domain: Bytes!
+  resolverEvents: [ResolverEvent!]! @derivedFrom(field: "resolver")
```

Comment:
Won't this return all events for this resolver contract, regardless of what domain they're for?

Reply:
see comment on line54